From: Dmitry Smirnov <onlyjob@member.fsf.org>
Date: Tue, 26 Mar 2013 22:00:33 +0100
Subject: path_to_cfg

Open .cfg file from where it is installed (not from CWD)

Forwarded: not-needed
---
 FreeOrion/client/human/chmain.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/FreeOrion/client/human/chmain.cpp b/FreeOrion/client/human/chmain.cpp
index 4a6ba8d..40e3917 100644
--- a/FreeOrion/client/human/chmain.cpp
+++ b/FreeOrion/client/human/chmain.cpp
@@ -215,7 +215,7 @@ int mainSetupAndRunOgre() {
 
 #ifndef FREEORION_WIN32
         log_manager->createLog((GetUserDir() / "ogre.log").string(), true, false);
-        root = new Root((GetRootDataDir() / "ogre_plugins.cfg").string());
+        root = new Root("/usr/lib/freeorion/ogre_plugins.cfg");
         // this line is needed on some Linux systems which otherwise will crash with
         // errors about GLX_icon.png being missing.
         Ogre::ResourceGroupManager::getSingleton().addResourceLocation((ClientUI::ArtDir() / ".").string(),
